userdocs/src/usage/eksctl-karpenter.md

```diff
@@ -34,8 +34,9 @@ to be set:
 
 ```yaml
 karpenter:
-  version: '0.4.3'
-  createServiceAccount: true # default is false  
+  version: '0.6.0'
+  createServiceAccount: true # default is false
+  defaultInstanceProfile: 'KarpenterNodeInstanceProfile' # default is to use the IAM instance profile created by eksctl
```

Comment:
This is great! But, also please add `-cluster-with-karpenter` to signify that the role should be postfixed with the cluster name so other clusters with Karpenter don't get into a naming conflict. :)